src/watchdog_ioctl.c

```diff
@@ -26,7 +26,7 @@ watchdog_ioctl(struct tcb *const tcp, const unsigned int code,
 	switch (code) {
 	case WDIOC_GETSUPPORT:
 		if (entering(tcp)) {
-			tprint_arg_next();
+			tprints_arg_next_name("argp");
 			return 0;
 		}
```

Comment:
Let's move this `tprints_arg_next_name("argp");` invocation after the `entering` branch.